src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {RateLimiter} from 'ccip/libraries/RateLimiter.sol';
+import {TokenAdminRegistry} from 'ccip/tokenAdminRegistry/TokenAdminRegistry.sol';
+import {UpgradeableBurnMintTokenPool} from 'ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol';
+import {UpgradeableTokenPool} from 'ccip/pools/GHO/UpgradeableTokenPool.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {GovernanceV3Arbitrum} from 'aave-address-book/GovernanceV3Arbitrum.sol';
+import {MiscArbitrum} from 'aave-address-book/MiscArbitrum.sol';
+import {IGhoToken} from 'gho-core/gho/interfaces/IGhoToken.sol';
+import {IUpgradeablePool} from 'src/interfaces/ccip/IUpgradeablePool.sol';
```

Comment:
pls use the full name `IUpgradeableTokenPool`, i would recommend it defined [like this](https://github.com/aave/aave-proposals-v3/blob/feat/ccip_1_5_1/src/interfaces/ccip/tokenPool/IUpgradeableLockReleaseTokenPool.sol) - you can re-use these interfaces and would probably just need to add an for ProxyPool's 1.5 style `applyChainUpdate`

Reply:
I renamed in [4398967](https://github.com/aave/aave-proposals-v3/commit/43989670344718a74d7fd9092ed7512e1d15455a). I personally like the 2 separate files for each version. I also have a to-do to remove dependencies and use minimal interfaces